src/groovy/org/pih/warehouse/importer/CSVUtils.groovy

```diff
@@ -43,7 +43,7 @@ class CSVUtils {
         if (StringUtils.isBlank(s)) {
             throw new IllegalArgumentException("${fieldName} is empty or unset")
         }
-        DecimalFormat format = DecimalFormat.getNumberInstance(LocalizationUtil.localizationService.currentLocale)
+        DecimalFormat format = DecimalFormat.getNumberInstance(LocalizationUtil.localizationService.currentLocale) as DecimalFormat
```

Comment:
Can you explain why the coersion is necessary?

Reply:
resolved in 3ddb00d5c